src/staging/gatsby-node.js

```diff
@@ -351,15 +356,11 @@ async function createSites({ graphql, actions, reporter }) {
   const result = await strapiApiRequest(graphql, siteQuery)
 
   result.data.allStrapiSites.nodes.forEach(site => {
-    // TODO: slug can be deleted when site.slug has created on all strapi env
-    const slug = slugify(site.siteName).toLowerCase()
-    const parkPath = site.protectedArea?.urlPath
-    // TODO: change sitePath `${parkPath}/${slug}` to `${parkPath}/${site.slug}` when site.slug has created on all strapi env
-    let sitePath = `${parkPath}/${slug}`
-    // If site doesn't have a relation with protectedArea, its path will be `parks/protected-area/${slug}`
-    if (!parkPath) {
-      sitePath = `parks/protected-area/${slug}`
-    }
+    // fallback in case site doesn't have a slug
+    const slug = site.slug ?? slugify(site.siteName).toLowerCase()
+    // fallback in case site doesn't have a relation with protectedArea
+    const parkPath = site.protectedArea?.urlPath ?? "parks/protected-area"
```

Comment:
Park urls don't use the pattern parks/golden-ears anymore.  They have been changed to `/golden-ears-park` or `/gitnadoiks-river-protected-area`.  Can the fallback be updated to follow the new pattern as well?   e.g. maybe `/no-protected-area`?